test/e2e/telemeter_test.go

```diff
@@ -21,30 +21,18 @@ import (
 	"testing"
 	"time"
 
-	"github.com/openshift/cluster-monitoring-operator/test/e2e/framework"
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 // TestTelemeterRemoteWrite verifies that the monitoring stack can send data to
 // the telemeter server using the native Prometheus remote write endpoint.
 func TestTelemeterRemoteWrite(t *testing.T) {
-	cm := &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      clusterMonitorConfigMapName,
-			Namespace: f.Ns,
-			Labels: map[string]string{
-				framework.E2eTestLabelName: framework.E2eTestLabelValue,
-			},
-		},
-		Data: map[string]string{
-			"config.yaml": "{}",
-		},
-	}
-	f.MustCreateOrUpdateConfigMap(t, cm)
+	cmoConfigMap := f.BuildCMOConfigMap(t, "{xxx}")
```

Comment:
the typo is on purpose to verify that BuildCMOConfigMap() fails if the input isn't valid.